### PR TITLE
CDAP-2795 Drop support for HBase 0.94

### PR DESCRIPTION
**Pull request by ghelmling**

### CODE REVIEW

File: cdap-watchdog/pom.xml [hunk 1]
```diff
@@ -70,6 +70,27 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-common</artifactId>
+      <scope>provided</scope>
```
Comment (gsps1) [inline cdap-watchdog/pom.xml]: scope is provided?

Comment (gsps1) [inline cdap-watchdog/pom.xml]: also why do we need this new dependencies in watch-dog ?

**Comment by gsps1** [comment]
just one question about new dependencies added. otherwise lgtm.

File: cdap-watchdog/pom.xml (hunk 1 above)
Comment (ghelmling) [inline cdap-watchdog/pom.xml]: Yes, we use provided scope for all HBase dependencies.  Prior to this it was pulling in HBase as a transitive dependency from elsewhere.

Comment (ghelmling) [inline cdap-watchdog/pom.xml]: Without this, you'll see compilation errors like:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project cdap-watchdog: Compilation failure: Compilation failure:
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[47,31] package org.apache.hadoop.hbase does not exist
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[48,38] package org.apache.hadoop.hbase.client does not exist
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[184,7] cannot find symbol
[ERROR] symbol:   class HBaseAdmin
[ERROR] location: class co.cask.cdap.metrics.store.upgrade.MetricsDataMigrator
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[184,32] cannot find symbol
[ERROR] symbol:   class HBaseAdmin
[ERROR] location: class co.cask.cdap.metrics.store.upgrade.MetricsDataMigrator
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[185,12] cannot find symbol
[ERROR] symbol:   class HTableDescriptor

**Comment by gsps1** [comment]
LGTM

**Pull request merged**